server/engine/resolver_test.go

```diff
@@ -16,7 +16,10 @@ var outputs map[string]map[string]interface{} // this will contain output values
 func TestResolveReferencesToParameters(t *testing.T) {
 	resolver := NewResolver(SUB_ID, RG_NAME)
 	resolver.ResolveReferencesToParameters(parameters, values)
-	resolver.ResolveReferencesToOutputs(parameters, outputs)
+	err := resolver.ResolveReferencesToOutputs(parameters, outputs)
+	if err != nil {
+		t.Errorf("Error while calling resolve outputs: %v", err)
+	}
```

Comment:
I think this should result in failing the test.

Reply:
t.Errorf does fail the test.